src/lib/operators/table_scan.hpp

```diff
@@ -22,6 +22,11 @@ class TableScan : public AbstractReadOnlyOperator {
 
   ~TableScan();
 
+  /**
+   * If set, the specified chunks will not be scanned.
```

Comment:
Can we have a comment somewhere why this is needed and why some operators have `set_included_chunk_ids` and others have excluded? That comment should be referenced here.